src/Providers/DataGrid/Wijmo/Features/ColumnPicker.ts

```diff
@@ -232,6 +266,9 @@ namespace Providers.DataGrid.Wijmo.Feature {
 
 				e.preventDefault();
 			};
+
+			// This code fixes WJM-34234
+			this._theColumnPicker.hostElement.addEventListener('keydown', this._issueWorkaround.bind(this));
```

Comment:
don't we need to remove this event somewhere?

Reply:
This was something that I didn't pay attention before, but I just found out the this._theColumnPicker.dispose() already takes care of it according to the [documentation](https://developer.mescius.com/wijmo/api/classes/Wijmo_Input.Listbox.html#dispose)